compileopts/target.go

```diff
@@ -271,3 +272,16 @@ func defaultTarget(goos, goarch, triple string) (*TargetSpec, error) {
 	}
 	return &spec, nil
 }
+
+func (spec *TargetSpec) Debugger() (string, error) {
```

Comment:
Can you rename this to `GDB`?

```suggestion
func (spec *TargetSpec) GDB() (string, error) {
```

We might want to add support for other debuggers in the future, such as LLDB (which can possibly be included in the TinyGo release, unlike GDB). LLDB will need special support, it is not a drop-in replacement.

Reply:
If I rename it to GDB(), I get the following error.
Is there any other name that would be better?

```
compileopts\target.go:276:6: type TargetSpec has both field and method named GDB
```

Reply:
Ah, I see. I can't think of a good name, maybe `GDBCommand` or `LookupGDB`?

Reply:
I renamed Debugger() to LookupGDB().